src/DemographicData.cpp

```diff
@@ -12,12 +16,37 @@
 
 using namespace amrex;
 
-DemographicData::DemographicData (const::std::string& fname)
+/*! Initializes by reading in demographic data from a given
+    filename. Calls DemographicData::InitFromFile(). */
+DemographicData::DemographicData (const::std::string& fname /*!< Name of file containing demographic data */)
 {
     InitFromFile(fname);
 }
 
-void DemographicData::InitFromFile (const std::string& fname)
+/*! \brief Read in demographic data from given file.
+ *
+ *  + The first line of the file contains the number of units.
+ *  + The following lines have the following data:
+ *    + ID (QDG: what is this ID?)
```

Comment:
@atmyers @tannguyen153 What is the first column of the demographic data file? Also, is my understanding of the columns (as described here) correct?